Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -13920,7 +13920,7 @@
 				);
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				INFOPLIST_FILE = "Wikipedia Stickers/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
After sending off to TestFlight we received an email with this warning:

"Architecture incompatible with MinimumOSVersion. The app bundle at 'Wikipedia.app/PlugIns/Wikipedia Stickers.appex' specifies a MinimumOSVersion of '12.0' but contains a 32-bit architecture that is unsupported on iOS 12 and later."

This fixes that. We had changed it to 12.0 originally because the project settings automatic update recommended it after upgrading Xcode.